cards.yaml

```diff
@@ -391,3 +391,72 @@ munihac2020:
     - using indecipherable, non-standard notation to hide a dodgy proof
     - wearing a T-shirt with a Linux joke
     - a slide deck entirely in Comic Sans
+
+cspapers:
+  black:
+    - CS paper: _ is morally correct
```

Comment:
This set looks good!  There's a yaml error though: because of the `:` this is parsed as an object:

```yaml
cspapers:
  black:
    - CS paper:
        _ is morally correct
```

So we need to quote to string:

```yaml
cspapers:
  black:
    - 'CS paper: _ is morally correct'
```